app/javascript/src/components/Common/DeleteAlert.jsx

```diff
@@ -0,0 +1,21 @@
+// @ts-nocheck
+import React from "react";
+
+import { Alert } from "neetoui";
+
+function DeleteAlert({ onClose, handleDelete }) {
+  return (
+    <div className="w-full">
+      <Alert
+        size="sm"
+        isOpen
```

Comment:
Implicitly true boolean props ought to be mentioned as the starting props within the list of props.

https://www.bigbinary.com/react-best-practices/miscellaneous-tips#declaring-boolean-props